.changes/v3.8.0/901-notes.md

```diff
@@ -0,0 +1,2 @@
+* VM Creation code refactored which should result in identifiable parts creation for all types of
+  VMs [GH-901]
```

Comment:
I would like to ask to clarify this statement. It's not very clear if you read it without deep knowledge of the topic. Don't hesitate to extend and elaborate with more details - it is critical piece of code after all.

Reply:
I have attempted to improve it by listing.